src/main/functions/DIDResolution.ts

```diff
@@ -23,7 +24,3 @@ export function getResolver(opts: ResolveOpts): Resolvable {
   }
   return new Resolver(...uniResolvers);
 }
-
-export async function resolveDidDocument(did: string, opts?: ResolveOpts): Promise<DIDDocument> {
```

Comment:
Is there a specific reason this is removed. Seem pretty useful to me

Reply:
added back